plugin/Makefile

```diff
@@ -60,10 +60,12 @@ PLUGIN_TAG := $(VERSION_TAG)$(EXTRA_TAG)
 
 # plugin name - used as a base for full plugin name and container for extracting rootfs
 PLUGIN_NAME=$(DOCKER_HUB_REPO)/docker-volume-vsphere
+SHARED_PLUGIN_NAME=$(DOCKER_HUB_REPO)/docker-volume-shared
```

Comment:
Well, to  be super accurate it needs to be docker-volume-vsphere-shared but it's a mouthful. 
maybe simply vsphere-shared ? "docker-volume-" seems redundand